data/settings/src/main/java/com/google/jetpackcamera/settings/model/CameraAppSettings.kt

```diff
@@ -24,7 +26,17 @@ data class CameraAppSettings(
     val front_camera_available: Boolean = true,
     val back_camera_available: Boolean = true,
     val dark_mode_status: DarkModeStatus = DarkModeStatus.SYSTEM,
-    val flash_mode_status: FlashModeStatus = FlashModeStatus.OFF
+    val flash_mode_status: FlashModeStatus = FlashModeStatus.OFF,
+    val aspect_ratio: Int = 1
 )
 
-val DEFAULT_CAMERA_APP_SETTINGS = CameraAppSettings()
\ No newline at end of file
+val DEFAULT_CAMERA_APP_SETTINGS = CameraAppSettings()
+
+fun getAspectRatioRational(aspectRatio: Int): Rational {
+    return when (aspectRatio) {
+        1 -> Rational(3,4)
+        2 -> Rational(9, 16)
+        3 -> Rational(1, 1)
+        else -> Rational(1, 1)
+    }
```

Comment:
I think it would be better to turn Aspect Ratio into an enum class with a constructor for the rational value. Put it under `data/settings/src/main/java/com/google/jetpackcamera/settings/model` with the other data enums